conda-forge.yml

```diff
@@ -1,6 +1,5 @@
 build_platform:
   linux_aarch64: linux_64
-  linux_ppc64le: linux_64
```

Comment:
Attempting to add `linux_ppc64le` cross-compilation in PR: https://github.com/conda-forge/implicit-feedstock/pull/91